cli/azd/internal/cmd/add/add_configure.go

```diff
@@ -142,6 +144,31 @@ func fillAiModelName(
 	return r, nil
 }
 
+func fillAiFoundryName(
+	_ context.Context,
+	r *project.ResourceConfig,
+	_ input.Console,
+	pOptions PromptOptions) (*project.ResourceConfig, error) {
+	if r.Name != "" {
+		return r, nil
+	}
+
+	// provide a default suggestion using the underlying model name
+	defaultName := "ai-models"
+	i := 1
+	for {
+		if _, exists := pOptions.PrjConfig.Resources[defaultName]; exists {
```

Comment:
It's also okay to just error out and validate for a single resource. I think N cardinality given resource type is a deeper problem to solve right now.